util/util.js

```diff
@@ -5,16 +5,21 @@
 }(this, function () { 
 	var util = {};
 
+	util.isValidObjectId = function(id){
+		var checkForHexRegExp = new RegExp("^[0-9a-fA-F]{24}$");
```

Comment:
if this check is for mongoose object id, I would like to say you 'd better let mongoose to do it, do not create your own version. If not ignore my comment.